daml/splice-amulet/daml/Splice/AmuletRules.daml

```diff
@@ -1283,6 +1312,8 @@ data TransferSummary = TransferSummary with
     amuletPrice : Decimal -- ^ The amulet price at the round this transfer was executed.
     inputValidatorFaucetAmount : Optional Decimal
       -- ^ Added in CIP-3. Total amount of validator faucet coupon issuance input into this transfer.
+    inputUnclaimedActivityRecordAmount : Optional Decimal
```

Comment:
@jose-velasco-ieu: am I correct, that the reason for `Optional` here is because otherwise the package upload complains due to a non-SCU compatible upgrade?

 @moritzkiefer-da: this seems like a case where SCU should stop inferring types as serializable and instead require explicit annotations. Otherwise internal types and computations using them become unnecessarily complicated. Do you see that the same?

Reply:
This here is the `TransferInputsSummary`, which is not referenced from the `TransferSummary`.

Reply:
ah sorry GH diff tripped me up. Then yes, you shouldn't need it. I don't tctually remember what the upgrade checks do here.